build/lib/i18n.ts

```diff
@@ -652,7 +652,15 @@ export function createXlfFilesForCoreBundle(): ThroughStream {
 			if (file.isBuffer()) {
 				const xlfs: Map<XLF> = Object.create(null);
 				const json: BundledFormat = JSON.parse((file.contents as Buffer).toString('utf8'));
-				for (let coreModule in json.keys) {
+				// {{SQL CARBON EDIT}} - Must sort the keys for easier translation, (Map class used here is custom, cannot sort keys directly).
```

Comment:
What type is json.keys? Can you use Object.keys(...) to get an array of the keys instead to simplify this a bit?

Reply:
That worked, thanks for reminding me of that function, that was what I had in mind.